admin/onpage/class-onpage-option.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * This class handles the data for the option where the OnPage.org data is stored.
+ */
+class WPSEO_OnPage_Option {
+
+	/**
+	 * @var array The OnPage.org option stored in the database.
+	 */
+	private $onpage_option;
+
+	/**
+	 * Setting the object by setting the properties
+	 */
+	public function __construct() {
+		$this->onpage_option = $this->get_option();
```

Comment:
We don't have to cache WordPress options, if we use `get_option` or `get_site_option` it is already cached by WordPress using `wp_cache_get`.